app/queries/hearing_request_distribution_query.rb

```diff
@@ -69,14 +77,34 @@ def most_recent_hearings
       joins(query, hearings: :hearing_day)
     end
 
+    def with_assigned_distribution_task_sql
+      # both `appeal_type` and `appeal_id` necessary due to composite index
+      <<~SQL
+        INNER JOIN tasks AS t1
```

Comment:
Let's be more descriptive esp. since `t1` is referenced outside this method
```suggestion
        INNER JOIN tasks AS distribution_task
```